service/forward_service.cc

```diff
@@ -358,9 +362,9 @@ future<query::forward_result> forward_service::dispatch_to_shards(
 
     forward_aggregates aggrs(req);
     co_return co_await aggrs.with_thread_if_needed([&aggrs, results = std::move(results), result = std::move(result)] () mutable {
-        for (auto& r : results) {
+        for (auto&& r : results) {
```

Comment:
Note: this change is meaningless, `auto&&` means use an lvalue if possible, otherwise rvalue. And lvalue is possible here.